FIRO_TSEnsembles/src/main/java/hec/ensemble/stats/NDayMultiComputable.java

```diff
@@ -20,8 +20,8 @@ public NDayMultiComputable(MultiComputable stepOne, float[] numberDays) {
     public float compute(float[] values) {
         values = stepOneCompute.multiCompute(values);
 
-        int timeStep = (int) config.getDuration().getSeconds();
-        int timeStepDay = 86400 / timeStep;
+        int timeStepSeconds = (int) config.getDuration().getSeconds();
+        int timeStepDay = 86400 / timeStepSeconds;  // How many time steps per day (86400 is how many seconds in a day).
         float interpVal = timeStepDay * days[0];
```

Comment:
Would the zero index correction make more sense here?
```suggestion
        float interpVal = timeStepDay * days[0] -1;
```